scripts/globals/magic.lua

```diff
@@ -1313,6 +1335,11 @@ function calculateDuration(duration, magicSkill, spellGroup, caster, target, use
         -- Gear mods
         duration = duration + duration * caster:getMod(dsp.mod.ENH_MAGIC_DURATION) / 100
 
+        -- prior according to bg-wiki
+        if caster:getMainJob() == dsp.job.RDM and caster:getMainLvl() >= 75 then
```

Comment:
Remove main job and main level check. They are not needed. getMerit handles this.